packages/connect/package.json

```diff
@@ -68,7 +68,8 @@
         "@types/karma": "^6.3.3",
         "@types/parse-uri": "^1.0.0",
         "@types/randombytes": "^2.0.0",
-        "jest": "^26.6.3",
+        "jest": "^29.5.0",
+        "jest-environment-jsdom": "^29.5.0",
```

Comment:
this is probably needed to install as long as there are other jest versions present